src/reducers/movieRatings.js

```diff
@@ -0,0 +1,10 @@
+export const movieRatings = (state = [], action) => {
+
+  switch (action.type) {
+    case 'SUBMIT_RATING':
+      return [...state, action.rating]
+    default:
+      return state;
+  }
+
+}
```

Comment:
I think we had removed this reducer because we are not using it in the App. Are we using it again?

Reply:
Good catch, Tim. I have this removed on my version as well.

Reply:
Weird. I thought I pulled down and merged the master. I'll go back and fix it.

Reply:
Merging isn't fixing it, so I'm just going to manually delete the file from this branch.

Reply:
Is there anything else weird like this that you all are seeing?

Reply:
Nope, all else looks good! I'll merge now.

Reply:
Everything else looks fine. I just notice that one code sneak backed in.  Overall, the testing looks great!